ws-client/src/client.rs

```diff
@@ -550,6 +587,20 @@ async fn background_task(
 					stop_subscription(&mut sender, &mut manager, unsub).await;
 				}
 			}
+
+			// User called `on_notification` on the front-end.
+			Either::Left((Some(FrontToBack::OnNotification(sub)), _)) => {
+				log::trace!("[backend] registering notification handler: {:?}", sub.sub_id);
+				let (subscribe_tx, subscribe_rx) = mpsc::channel(max_notifs_per_subscription);
+
+				if manager.insert_notification_handler(sub.req_id, sub.sub_id.clone(), subscribe_tx).is_ok() {
+					sub.send_back
```

Comment:
This expect should be removed because if the user drops the notification handle to this will panic,

so just ignore the error here by `let _ = send(...)`

Reply:
Are you sure about this one?  This insert should happen in the code path for register_notification() and that async is blocked on this send (or it should fail to register).

Reply:
That's a good comment but the reason is that the user may drop the `register_notifiction future` which is possible and then this will panic AFAIU ^^

Probably quite unlikely but better be safe than sorry I guess

Reply:
👍  ah that does make sense